securedrop/secure_tempfile.py

```diff
@@ -47,7 +48,12 @@ def __init__(self, store_dir):
         """
         self.last_action = 'init'
         self.create_key()
-        self.tmp_file_id = base64.urlsafe_b64encode(os.urandom(32)).decode('utf-8').strip('=')
+        self.temp_file_id = ""
+        if six.PY2:
```

Comment:
As in the other commend, we duplicate logic, and may make sense to use a more minimal branching on the python2/3 code.

Reply:
Doing this.